packages/protocol/test/governance/validators.ts

```diff
@@ -2171,7 +2171,7 @@ contract('Validators', (accounts: string[]) => {
         assert.equal(parsedValidator.affiliation, NULL_ADDRESS)
       })
 
-      it('should succeed when the sender is the governance contract', async () => {
+      it('should succeed when the sender is the governance slasher contract', async () => {
```

Comment:
I think we were thinking that this would stay `Governance`, that way someone could be slashed via governance without necessarily being removed from their group